core/src/main/resources/org/broadinstitute/dsde/rawls/liquibase/changesets/20210111_retroactive_default_cromwell_backend_per_project.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="gsterin" id="retroactive-default-cromwell-backend-per-project">
+        <sql stripComments="true">
+            UPDATE BILLING_PROJECT bp SET bp.CROMWELL_BACKEND = "PAPIv2" WHERE bp.CROMWELL_BACKEND IS NULL
```

Comment:
TOL was this supposed to be set in the workspace too?

Reply:
`WORKSPACE` table doesn't have such column

Reply:
Yeah that's what I was thinking. 🙂 And since this is the backfill part and we don't have projects per workspace yet this isn't a concern here?

Reply:
Yes, I think so